src/sentry/static/sentry/app/components/tooltip.tsx

```diff
@@ -213,17 +226,18 @@ class Tooltip extends React.Component<Props, State> {
               aria-hidden={!isOpen}
               ref={ref}
               style={style}
-              hide={!title}
+              hide={!title && !tipContent}
               data-placement={placement}
               popperStyle={popperStyle}
               onMouseEnter={() => isHoverable && this.handleOpen()}
               onMouseLeave={() => isHoverable && this.handleClose()}
             >
-              {title}
+              {tipContent ? tipContent : title}
```

Comment:
Do we need `tipContent`? Can't we use `title` prop?

Reply:
Right, I will